setup.py

```diff
@@ -93,6 +94,10 @@ def run(self):
         'console_scripts': [
             'jupyter-kernelspec = jupyter_client.kernelspecapp:KernelSpecApp.launch_instance',
             'jupyter-run = jupyter_client.runapp:RunApp.launch_instance',
+        ],
+        'jupyter_client.kernel_providers' : [
+            'spec = jupyter_client.discovery:KernelSpecProvider',
+            'pyimport = jupyter_client.discovery:IPykernelProvider',
```

Comment:
Are we being consistent with the id before = as described in entrypoints doc section.

Reply:
Yep:

https://github.com/takluyver/jupyter_client/blob/c09b8aced5912901b4e96e11d9d4eeb3346b4fd7/jupyter_client/discovery.py#L30

https://github.com/takluyver/jupyter_client/blob/c09b8aced5912901b4e96e11d9d4eeb3346b4fd7/jupyter_client/discovery.py#L55

I can't work out when Github does inline code previews and when it doesn't!